src/dfcx_scrapi/core/conversation.py

```diff
@@ -301,23 +303,80 @@ def _page_id_mapper(self):
 
         self.agent_pages_map = agent_pages_map.reset_index(drop=True)
 
-    def _get_reply_results(self, utterance, page_id, results, i):
+    def _get_reply_results(
```

Comment:
let's revert these changes here for this PR

Reply:
reverted